packages/webapp/src/elections/components/ongoing-election-components/meeting-link/meeting-link.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from "react";
+import { Dayjs } from "dayjs";
+
+import {
+    delay,
+    encryptSecretForPublishing,
+    onError,
+    useEncryptedData,
+    useMemberGroupParticipants,
+    useUALAccount,
+    useVoteDataRow,
+    useZoomAccountJWT,
+} from "_app";
+import { generateZoomMeetingLink } from "_api/zoom-commons";
+import { setElectionMeeting } from "elections/transactions";
+import { calculateGroupId } from "elections/utils";
+import { ActiveStateConfigType, RoundStage } from "elections/interfaces";
+
+import MeetingButtons from "./meeting-buttons";
+import MeetingLinkModal from "./meeting-modal";
+
+export enum MeetingStep {
+    LinkZoomAccount,
+    CreateMeetingLink,
+    RetrieveMeetingLink,
+}
+
+interface RequestMeetingLinkProps {
```

Comment:
nit: rename to `MeetingLinkProps` or just plain `Props` since we know this is the props for this current component.